packages/foam-core/src/markdown-provider.ts

```diff
@@ -380,10 +378,7 @@ export function createMarkdownParser(
               }
             }
           } catch (e) {
-            Logger.warn(
-              `Error while parsing YAML for [${URI.toString(uri)}]`,
-              e
-            );
+            Logger.warn(`Error while parsing YAML for [${uri}]`, e);
```

Comment:
how come removing the `toString` from the logging statements?

Reply:
I accidentally reverted to the wrong commit. I thought I fixed it, but I was struggling with git.